### PR TITLE
made contributors.md

### LINKED ISSUE #1028: [Documentation] Create Contributors.md file
## Description
Currently we don't any separate file in the repo showing the list of contributors.

### Update
- [ ] Create `Contributors.md` file that will contain all the basic information of all the contributors of this repo. along with their images.

## Definition of Done
- [ ] All of the required items are completed.
- [ ] Approval by 1 mentor.

## Estimation
0.5 hours

### PR DESCRIPTION
**Pull request by vaibhavarora102**
# Description
added the contributors.md file and installed all-contributors bot as a part of the open-source hack
Fixes  #1028

# Type of Change:
**Delete irrelevant options.**

- Documentation

# How Has This Been Tested?
just added a readme file

# Checklist:
**Delete irrelevant options.**

- [/ ] My PR follows the style guidelines of this project
- [/ ] I have performed a self-review of my own code or materials
- [ /] I have made corresponding changes to the documentation
- [ /] Any dependent changes have been merged  (no dependencies)

### CODE REVIEW

**Comment by Kajol-Kumari** [comment]
Hey @vaibhavarora102 thanks for the PR but I think this bot won't work untill we provide them the read and write access of the `vms` main repo. So, I am currently looking for a way to implement it. Meanwhile you can work on the other issue.

**Comment by vaibhavarora102** [comment]
Oh thanks for information

**Review by Kajol-Kumari — CHANGES_REQUESTED** [review]
@vaibhavarora102 as you can't install bot due to permission requirement. please go for adding the contributors list in the `contributors.md` file manually. We will install bot for the repo later on.

**Comment by vaibhavarora102** [comment]
Can you pls tell the format in which I had to write

**Comment by Kajol-Kumari** [comment]
> Can you pls tell the format in which I had to write

@vaibhavarora102 keep it in tabular form having details like contributor's name, their github profile link and the number of commits they have made.

**Comment by vaibhavarora102** [comment]
Should I close this pr and add another one?

**Comment by vaibhavarora102** [comment]
okay

**Comment by vaibhavarora102** [comment]
hi @Kajol-Kumari I had done all my work 
kindly review it and approve for merge

File: Contributors.md
Comment (Kajol-Kumari) [inline Contributors.md:73]: please provide a space in `Thankyou` i.e make it like `Thank you`

**Review by Kajol-Kumari — CHANGES_REQUESTED** [review]
please rename the file as `CONTRIBUTORS.md` as all the `.md` files have names in this format only

**Comment by vaibhavarora102** [comment]
@Kajol-Kumari  can you pls check now

**Review by Kajol-Kumari — DISMISSED** [review]
LGTM :+1:  
@vaibhavarora102 please squash the commits into a single commit so that it can be sent for final approval.

**Comment by vaibhavarora102** [comment]
Kindly now take it for final review

**Comment by Kajol-Kumari** [comment]
> Kindly now take it for final review
Hey @vaibhavarora102 I can see still 8 commits in this PR please squash them in a single one.

**Review by SanketDG — APPROVED** [review]
Looks good to me, was this done manually?

**Comment by vaibhavarora102** [comment]
Hi @Kajol-Kumari 
Can I expect now my pr to get merged soon

**Comment by SanketDG** [comment]
I don't like this being done manually, why not just use https://allcontributors.org/docs/en/cli/overview for now, and then we can see if we can get the bot integrated by some maintainer.

**Comment by Kajol-Kumari** [comment]
> I don't like this being done manually, why not just use https://allcontributors.org/docs/en/cli/overview for now, and then we can see if we can get the bot integrated by some maintainer.



> I don't like this being done manually, why not just use https://allcontributors.org/docs/en/cli/overview for now, and then we can see if we can get the bot integrated by some maintainer.

@SanketDG If we go on adding with this ^ then:
1. It will just add extra dev-dependency which we won't require after the bot installation.
2. I am not sure, if we would be able to create the contributor list using this CLI from a fork repo. for the main repo.using this ^

**Comment by Kajol-Kumari** [comment]
> Hi @Kajol-Kumari
> Can I expect now my pr to get merged soon

@vaibhavarora102 All the PRs having 2 or more approvals will be merged but only after the event ends. So, if your PR has got atleast two approvals, it's good to go and you can work on other PRs.

**Comment by vaibhavarora102** [comment]
Actually it was on my default branch 
So, I could only work freely once it's merged

And I think it's been approved by 2 times and it's a kind request to merge if everything is well

**Comment by Kajol-Kumari** [comment]
Hey @vaibhavarora102 we can't merge any of the PR before the end of the event and participants are always supposed to create a branch and then start working on an issue. SO i would recommend you to make a fresh fork, create a branch, make th erequired changes , close this Pr and open up a new one.

**Comment by SanketDG** [comment]
> 1. It will just add extra dev-dependency which we won't require after the bot installation.

I am not sure I follow, I am asking for the cli to be used once to generate the files in README.md and then send the PR.
Exactly what's being done now, except through a tool and not manually.

> 2\. I am not sure, if we would be able to create the contributor list using this CLI from a fork repo. for the main repo.using this ^

Again, I do not follow, it's just like changing a file and sending a PR right? The diff would be against README.md

---

Later we just integrate this with the bot itself (when a maintainer does this)

**Comment by Kajol-Kumari** [comment]
> > 1. It will just add extra dev-dependency which we won't require after the bot installation.
> 
> I am not sure I follow, I am asking for the cli to be used once to generate the files in README.md and then send the PR.
> Exactly what's being done now, except through a tool and not manually.
> 
> > 2. I am not sure, if we would be able to create the contributor list using this CLI from a fork repo. for the main repo.using this ^
> 
> Again, I do not follow, it's just like changing a file and sending a PR right? The diff would be against README.md
> 
> Later we just integrate this with the bot itself (when a maintainer does this)

@SanketDG I hadn't got your proposal properly. This sounds good to me. @vaibhavarora102 can you please integrate the CLI and create the contributor list using that only. Also as you have created this PR using your master branch. Please clone this one and make a new PR through a particular branch dedicated for this issue.

**Pull request closed without merging**
